gradle.properties

```diff
@@ -1,15 +1,11 @@
 projectVersion=5.3.0-SNAPSHOT
 projectGroup=io.micronaut.liquibase
-micronautDocsVersion=2.0.0
```

Comment:
The micronaut docs annotation processor is injected automagically by the common plugin I think.

So we need this here (as the plugin uses it for the version)

However we can remove both `micronaut-docs = "2.0.0"` and the `micronaut-docs = {` line from `libs.versions.toml`

And then we can remove the `annotationProcessor libs.micronaut.docs` from `liquibase/build.gradle`